src/posv_mixed.cc

```diff
@@ -168,22 +170,27 @@ void posv_mixed(
     copy( A, A_lo, opts );
 
     // Compute the Cholesky factorization of A_lo.
+    Timer t_potrf_lo;
     potrf(  A_lo, opts );
+    timers[ "posv_mixed::potrf_lo" ] = t_potrf_lo.stop();
 
     // Solve the system A_lo * X_lo = B_lo.
+    Timer t_potrs_lo;
     potrs( A_lo, X_lo, opts );
+    timers[ "posv_mixed::potrs_lo" ] = t_potrs_lo.stop();
 
     // Convert X_lo to high precision.
     copy( X_lo, X, opts );
 
     // Compute R = B - A * X.
     slate::copy( B, R, opts );
+    Timer t_hemm_lo;
     hemm<scalar_hi>(
         Side::Left,
         -one_hi, A,
                  X,
         one_hi,  R, opts );
-
+    timers[ "posv_mixed::hemm_lo" ] = t_hemm_lo.stop();
```

Comment:
Leave this blank line.

Reply:
Added a blank line after `hemm` for `posv_mixed.cc` and added a blank line after `gemm` for `gesv_mixed.cc`. Are you wanting me to add a space after each `hemm` or `gemm` operation? Or was this due to the following lines being `colNorms()`?

Reply:
Just, if there was a blank line already, please don't delete it.